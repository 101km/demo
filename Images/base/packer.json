{

"builders": [{
  "type": "amazon-ebs",
  "region": "us-west-2",
  "source_ami": "ami-d2c924b2",
  "instance_type": "t2.micro",
  "ssh_username": "centos",
  "ssh_pty" : false,
  "ami_name": "cncfbase{{timestamp}}",
  "user_data_file": "disable_tty.sh"
}],

"provisioners": [{
      "type": "ansible",
      "playbook_file": "playbook.yml",
      "user": "centos",
      "sftp_command": "/usr/libexec/openssh/sftp-server",
      "extra_arguments": [ "-vvv", "--extra-vars", "packer=yes" ]
  }]

}
