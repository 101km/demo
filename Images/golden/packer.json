{

"builders": [{
  "type": "amazon-ebs",
  "region": "us-west-2",
  "source_ami": "ami-8c53f1ec",
  "instance_type": "t2.micro",
  "ssh_username": "centos",
  "ssh_pty" : false,
  "ami_name": "cncf{{timestamp}}",
  "user_data_file": "disable_tty.sh"
}],

"provisioners": [{
      "type": "ansible",
      "playbook_file": "playbook.yml",
      "user": "centos",
      "sftp_command": "/usr/libexec/openssh/sftp-server",
      "extra_arguments": [ "--extra-vars", "packer=yes" ]
  }]

}
